src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala

```diff
@@ -0,0 +1,35 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
+import com.linagora.gatling.imap.protocol.Uid
+import com.linagora.gatling.imap.protocol.command.FetchAttributes.AttributeList
+import com.linagora.gatling.imap.protocol.command.FetchRange._
+import io.gatling.app.Gatling
+import io.gatling.core.Predef._
+import io.gatling.core.config.GatlingPropertiesBuilder
+import io.gatling.core.scenario.Simulation
+
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+
+class ImapSimpleScenario extends Simulation {
+  val UserCount: Int = 3
+  val feeder = (0 until UserCount).map(i => Map("username" -> s"user$i", "password" -> "password")).toArray
+
+  val scn = scenario("Imap").feed(feeder)
+    .pause(1 second)
+    .exec(imap("Connect").connect()).exitHereIfFailed
+    .exec(imap("login").login("${username}", "${password}").check(ok))
+    .during(1 minute) {
+        exec(imap("list").list("", "*").check(ok, hasFolder("INBOX")))
+        .pause(200 milli)
+        .exec(imap("select").select("INBOX").check(ok))
+        .pause(200 milli)
+        .exec(imap("fetch").fetch(Seq(Last()), AttributeList("BODY[HEADER]", "UID", "BODY[TEXT]")).check(no))
```

Comment:
> fetch should use a vararg

Not sure you can, as we are using an Expression[Seq[FetchRange]]

> and I'd rather say Range.Last() or range = Last()

of course you can, but for me Last() is crystal clear for a fetch command.